### DISCUSSION TITLE
SetVirtualHostNameToFolderMapping

### DISCUSSION THREAD

**Discussion by asv7c2** [MicrosoftEdge/WebView2Feedback#2045] [Q&A]
In [SetVirtualHostNameToFolderMapping](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2_3?view=webview2-1.0.1054.31&preserve-view=true#setvirtualhostnametofoldermapping) docs there is mention that media files accessed using virtual host name can be very slow to load. This is because it implement as non-async fileserver? Just suggestion.

**Comment by victorthoang** [comment]
hello @asv7c2,

Are you still looking for an answer to your question? If you still need a response, I can turn this into an Issue with a Question tag (which is typically how we address questions and bugs.

The discussion thread might be more useful for off topic questions or non-issue related inquiries!
